build.sbt

```diff
@@ -261,6 +266,22 @@ lazy val doc = project
     core
   )
 
+lazy val nodeFlywaySettings = {
+  lazy val DB_HOST = "localhost"
+  lazy val DB_NAME = "nodedb"
+  lazy val unittest = List(
+    Test / flywayUrl := s"jdbc:sqlite:${System.getenv("HOME")}/.bitcoin-s/unittest/nodedb.sqlite",
+    Test / flywayLocations := List("nodedb/migration"),
+    Test / flywayUser := "nodedb",
+    Test / flywayPassword := "",
+    flywayUrl := s"jdbc:sqlite:${System.getenv("HOME")}/.bitcoin-s/unittest/nodedb.sqlite",
+    flywayUser := "nodedb",
+    flywayPassword := ""
+  )
+
+  unittest
```

Comment:
I assume in the future we'll have other options for this with configurable url etc?

Reply:
Currently migrations are only going to affect this one database and it looks like there are others?